src/backend/distributed/connection/connection_management.c

```diff
@@ -1172,29 +1172,3 @@ TrimLogLevel(const char *message)
 
 	return chompedMessage + n;
 }
-
-
-/*
- * NodeConnectionCount gets the number of connections to the given node
- * for the current username and database.
- */
-int
-NodeConnectionCount(char *hostname, int port)
-{
-	ConnectionHashKey key;
-	ConnectionHashEntry *entry = NULL;
-	bool found = false;
-
-	strlcpy(key.hostname, hostname, MAX_NODE_LENGTH);
-	key.port = port;
-	strlcpy(key.user, CurrentUserName(), NAMEDATALEN);
-	strlcpy(key.database, CurrentDatabaseName(), NAMEDATALEN);
-
-	entry = (ConnectionHashEntry *) hash_search(ConnectionHash, &key, HASH_FIND, &found);
-	if (!found)
-	{
-		return 0;
-	}
-
-	return entry->connectionCount;
```

Comment:
I guess `entry->connectionCount` becomes unused with this change, so maybe remove?